tests/test_peak_detection.py

```diff
@@ -1336,3 +1338,14 @@ def test__D4_data_causing_TwoValleysInARowError(MA202000030__2020_12_11_233215__
 
     assert np.array_equal(peak_indices, expected_peak_indices)
     assert np.array_equal(valley_indices, expected_valley_indices)
+
+
+def test__A3_data_causing_TwoValleysInARowError(MA202000127__2021_03_26_174059__A3):
+    _, peak_and_valley_indices = MA202000127__2021_03_26_174059__A3
+    _, valley_indices = peak_and_valley_indices
+
+    # fmt: off
+    expected_valley_indices = [637, 1361, 2174,2933,3566,4335,5107,5863,6688,7411,7933,8711,9570,10364,11176,11768,12706,13420,14050,14852,15648,16355,17189,17954,18622,19402,20151,20785,21475,22437,23061,23882,24582,25284,26023,26627,27495,28254,29039, 29794,30533,31308,32015,32570,33518,34239,34957,35696,36327,36953,37930,38685,39342,40059,40816,41426,42374,43146,43829,44515,45326,46054,46725,47521,48253,48993,49730,50421,51157,51811,52497,53279,54052,54743,55549,56250,56885,57749,58513,59191,]
```

Comment:
note for future, you can just check a slice of the array to see if it matches (maybe the 3-4 positions around where you care about), that way the test is less brittle and you don't have to put so many numbers into the file